web/massastation/src/mirage/server.js

```diff
@@ -79,7 +111,26 @@ function mockServer(environment = ENV.DEV) {
         });
         return new Response(200, {});
       });
-
+      this.post('plugin-manager', (schema, request) => {
+        const sourceURL = request.queryParams.source;
+        const storePlugin = schema.stores.findBy(
+          (store) => store.file.url === sourceURL,
+        );
+        // use the parameters of the store plugin if it exists
+        // otherwise generate random values
+        const newPlugin = {
+          id: faker.number.int(),
+          status: 'Up',
+          updatable: false,
+          name: storePlugin?.name || faker.lorem.word(),
+          author: storePlugin?.author || faker.person.firstName(),
+          description: storePlugin?.description || faker.lorem.sentence(),
+          version: storePlugin?.version || faker.system.semver(),
+          home: storePlugin?.url || `/plugin/massalabs/${faker.lorem.word()}/`,
+        };
+        schema.plugins.create(newPlugin);
+        return new Response(204);
```

Comment:
```suggestion
        };
        
        return schema.plugins.create(newPlugin);
```

Reply:
nope, see https://github.com/massalabs/station/pull/799#discussion_r1234159547

Reply:
wouldn't this return the plugin ?

Reply:
According BE docs not. It's an empty 204 as you did! Good for me!